### DISCUSSION TITLE
update button is disabled when changing from hybrid to online-only

### DISCUSSION THREAD

**Discussion by YodelingCowboys** [code4recovery/12-step-meeting-list#640] [Q&A] [answered]
Never had a problem updating meetings before, but this is the first time I have been asked to change a meeting back to online-only.

I removed the location and address.  Then change attend inperson from yes to no.  The update button is disabled.

This is happening on three websites -- central office, intergroup, and area.

Both wordpress and TSML are at the latest versions.

Let me know what additional information you need.  I will not provide login credentials in this forum,  provide email address or phone number to contact.

**Comment by Grateful-Viv** [comment, marked as answer]
You shouldn’t need to remove the address. Simply change option from yes to no about meeting in person, and make sure you put a valid URL in the meeting link area.

**Comment by YodelingCowboys** [comment]
Thank you, Grateful-Viv.

When I tried just changing the attend inperson flag to no, I received this
message:
Warning: Online meetings with a specific address will appear that the
location temporarily closed. Meetings that are Online only should use
appoximate addresses.

Yesterday, when I saw the warning, I reacted to the unwanted "location
temporarily closed."  I failed to comprehend the second sentence.  I was
removing the entire address instead of just the street address.  Now
everything works fine.

It seems that I too often need arrogant first-reactions to remind me of
humility.  Again, Thank You!
